fedopt_guide/cifar10/federated_cifar10.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2019, Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Federated CIFAR-10 classification library using TFF."""
+
+import functools
+from typing import Callable, Optional
+
+from absl import logging
+import tensorflow as tf
+import tensorflow_federated as tff
+
+from utils import training_loop
```

Comment:
Let's use the training loop in the fedopt_guide folder. It's designed specifically for these tasks.